tbxforms/static/js/tbxforms.js

```diff
@@ -1,142 +1,166 @@
 import '../sass/tbxforms.scss';
 
-class TbxForms {
-  static selector() {
-    return 'form.tbxforms';
+/**
+ * CustomEvent polyfill for IE11.
+ * https://stackoverflow.com/a/26596324/1798491
+ */
+(function () {
+  if (typeof window.CustomEvent === 'function') return false; //If not IE
```

Comment:
If not IE, when is `CustomEvent` defined?

Reply:
This is a standard DOM API: https://developer.mozilla.org/en-US/docs/Web/API/CustomEvent